hooks/post-command

```diff
@@ -1,62 +1,44 @@
 #!/bin/bash
-# shellcheck disable=SC2001
-set -euo pipefail
+set -ue
 
-if [[ "${BUILDKITE_PLUGIN_CACHE_DEBUG:-false}" =~ (true|on|1) ]] ; then
-  set -x
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+# shellcheck disable=SC1090
+. "$DIR/../lib/shared.bash"
+
+# get bucket name from org slug ie.. chef-oss or chef
+if [[ "${BUILDKITE_ORGANIZATION_SLUG}" == "chef-oss" ]]; then
```

Comment:
I'm not sure we want this logic built into the plugin, especially if its open source. We probably want this logic to be handled by expeditor, and to have the bucket specified as a plugin parameter.

Reply:
okay, sounds good! I can get that updated in the expeditor code and have this pull in that parameter